pom.xml

```diff
@@ -166,6 +166,30 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.8.7</version>
+        <configuration>
```

Comment:
you do not need this section let's skip it

Reply:
Done